components/ResultsPage/YourAnswers.tsx

```diff
@@ -41,6 +42,20 @@ export const YourAnswers: React.VFC<{
     }))
   }
 
+  /**
+   * Deducts $5,000 from the first $5,000 and 50% of the next $10,000.
+   * The maximum deduction is $10,000 for inputs of $15,000 or more.
+   */
+  function salaryExemption(income: number): number {
+    if (income <= 5000) {
+      return income
+    } else if (income <= 15000) {
+      return 5000 + (income - 5000) * 0.5
+    } else {
+      return 10000
+    }
+  }
+
```

Comment:
"Your answers" should just capture inputs from the questions page, why the income logic here?

Reply:
The "What you've told us" section has a new field called Salary exemption, this field does not display the original input value for `incomeWork` field, instead it displays the deducted value after calculation, so I added this to adjust the `incomeWork` value.